### DISCUSSION TITLE
Using non-harmonised versions and custom scores?

### DISCUSSION THREAD

**Discussion by scienception** [PGScatalog/pgsc_calc#401] [Q&A] [answered]
Is it ok to use the non-harmonized version of the PGS files? Or does `pgs_calc` expect exclusively this format?

![Screenshot 2024-12-18 at 15 11 42](https://github.com/user-attachments/assets/0d448f03-c2b2-4c9b-9c45-5dff9eada544)

I ask cause I developed my own PGS and don't have any info on `hm_source, hm_rsID, hm_chr, hm_pos or hm_inferOtherAllele`. Only on `rsID, chr_name, chr_position, effect_allele, other_allele, effect_weight.`

Will `pgs_calc` accept this format?

**Comment by nebfield** [comment, marked as answer]
You can use your own PGS by following this guide:

https://pgsc-calc.readthedocs.io/en/latest/how-to/calculate_custom.html#scorefile-setup

Your PGS scoring file needs a special header and some standard column names to be compatible
